accessibility-checker-engine/src/v2/aria/ARIAMapper.ts

```diff
@@ -683,7 +713,8 @@ export class ARIAMapper extends CommonMapper {
                 let nodeName = parent.nodeName.toLowerCase();
                 // If nearest sectioningRoot or sectioningContent is body
                 while (parent) {
-                    if (sectioningRoots[nodeName] || sectioningContent[nodeName]) {
+                    let role = (parent.nodeType === 1 && (parent as HTMLElement).getAttribute("role")) || ""
+                    if (sectioningRoots[nodeName] || sectioningContent[nodeName] || sectioningRole[role]) {
```

Comment:
Added in last 2 Commit see https://github.com/IBMa/equal-access/pull/541/commits/e5faef01aa09b64731d9089c96a820e14aea5c7e

Reply:
@tombrunet Confirm this looks fine to you. Since I this is not running with test cases against my rule. Fwiw it does seem to not break anything
![image](https://user-images.githubusercontent.com/4451128/145294268-fdf0c28a-e7d7-46c2-8e82-8c29c65ad97d.png)